lib/svg-preview-view.js

```diff
@@ -27,6 +27,17 @@ class SvgPreviewView extends View {
             this.text('transparent')
           })
         })
+        this.div({ class: 'image-controls-zoom-group' }, () => {
+          this.a({ outlet: 'zoomInButton', class: 'image-controls-zoom-in' }, () => {
+            this.text('+')
+          })
+          this.a({ outlet: 'zoomOutButton', class: 'image-controls-zoom-out' }, () => {
+            this.text('-')
+          })
+          this.a({ outlet: 'zoomResetButton', class: 'image-controls-zoom-reset' }, () => {
+            this.text('1:1')
+          })
+        })
```

Comment:
I used the code style from this package but the content from the reference you pointed to.

But I have no idea how the buttons should look like without custom icons / style. In the current state the functionality works but it does look pretty bad. Any advice how this should be changed?